### PR TITLE
Update Slovak translation.

### PR DESCRIPTION
**Pull request by jose1711**

### CODE REVIEW

**Comment by lurch** [comment]
There's a bunch of translations that you've updated, which still say `type="unfinished"` ?

**Comment by jose1711** [comment]
> There's a bunch of translations that you've updated, which still say `type="unfinished"` ?

You're right. Corrected now. Thank you for letting me know.

File: src/i18n/rpi-imager_sk.ts [hunk 1]
```diff
@@ -648,7 +648,7 @@
     <message>
         <location filename="../main.qml" line="500"/>
         <source>[ All ]</source>
-        <translation type="unfinished"></translation>
+        <translation>[ Všetky ]</translation>
```
Comment (tdewey-rpi) [inline src/i18n/rpi-imager_sk.ts:651]: I'm particularly impressed you translated this string - it's often omitted by translators.

**Review by tdewey-rpi — APPROVED** [review]
Dakujem, @jose1711 

Apologies this took so long to come to - but you'll see it in v1.9.1.

**Pull request merged**